docs/ensemble_scheduler.md

```diff
@@ -287,6 +298,19 @@ By analyzing logs and timestamps it is seen that googlenet and resnet model infe
 [2020-09-04 12:46:18.849] [serving] [info] [prediction_service_utils.cpp:59] Requesting model:argmax; version:0.
 ```
 
+### Step 6: Requesting pipeline metadata
+
+We can use the same gRPC/REST example client as we use for requesting model metadata. The only difference is we specify pipeline name instead of model name.
```

Comment:
> The only difference is we specify pipeline name instead of model name.
I am not sure if this is necessary comment.

What we may want to mention here is special handling of input/output that we do not have knowledge apriori (hence DT_INVALID, empty shape).